### DISCUSSION TITLE
Add get_annotated_regions_from_gb.py to getorganelle

### DISCUSSION THREAD

**Discussion by o-william-white** [Kinggerm/GetOrganelle#197] [Ideas]
Hello, 

Would it be possible to add the python script get_annotated_regions_from_gb.py in your personal utilities repo to the get organelle utilities? 

I am writing a galaxy wrapper for this script so that it can be used with getorganelle in galaxy. See PR here: https://github.com/galaxyproject/tools-iuc/pull/4783 

I can open at PR if helpfull

Best wishes
Ollie

**Comment by Kinggerm** [comment]
Hi, I've thought about it. It's not a bad idea. 

I can simply add it to GetOrganelle, giving the license is also compatible in this direction. 

However, I do not want to add its specific dependency BioPython to the GetOrganelle dependencies list, but want to leave it to users upon its usage. Please let me know if it makes sense.

**Reply by o-william-white** [reply]
Hi, that sounds like a good idea, thanks! If you let me know when it has been updated, I will adjust the Galaxy tool

**Reply by bernt-matthias** [reply]
Hi @Kinggerm 

I agree. That this would work. We can add biopython as extra dependency to the Galaxy tool. We just would need a new release containing the python script.

**Reply by Kinggerm** [reply]
I have added it to GetOrganelle GitHub. However, I need a little more time to test other updates before drafting the release. I will keep you guys posted.
